Tasks/Common/webdeployment-common-v2/ziputility.ts

```diff
@@ -25,13 +48,32 @@ const extractUsingUnzip = async (fromFile: string, toDir: string) => {
         .exec();
 }
 
+const isMSBuildPackageZip = async (packageZipPath: string): Promise<boolean> => {
+    let pacakgeComponent = await getArchivedEntries(packageZipPath);
+    if (((pacakgeComponent["entries"].indexOf("parameters.xml") > -1) || (pacakgeComponent["entries"].indexOf("Parameters.xml") > -1)) && 
```

Comment:
here, won't it be better if we just iterate through the items in the 'entries' array just once and do a case invariant match for parameters.xml and systeminfo.xml. Right now, each indexOf will iterate through the array once and we might miss a match due to Uppercase/Lowercase mismatch